pkg/controller/ps/telemetry.go

```diff
@@ -0,0 +1,123 @@
+package ps
+
+import (
+	"context"
+	"fmt"
+
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/telemetry"
+	"github.com/pkg/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func (r *PerconaServerMySQLReconciler) reconcileScheduledTelemetrySending(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL) error {
+	logger := logf.FromContext(ctx).WithName("reconcileScheduledTelemetrySending")
```

Comment:
this names for logs usually pollute the logs, i think we should remove it

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/1033/commits/01f6c0a824538246434f78e01949ead0ae983838